package.json

```diff
@@ -93,6 +93,7 @@
     "randombytes": "^2.1.0",
     "store2": "2.14.2",
     "stream-browserify": "3.0.0",
+    "web3": "^1.8.2",
```

Comment:
I think dev_dependencies is the right place
Maybe look also for some of the others if they are at right place (stream-browserify ??)